cmd/openshift-install/main.go

```diff
@@ -1,121 +1,51 @@
 package main
 
 import (
-	"fmt"
-	"os"
-	"os/exec"
-	"strings"
-
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"gopkg.in/alecthomas/kingpin.v2"
-
-	"github.com/openshift/installer/pkg/asset"
-	"github.com/openshift/installer/pkg/asset/cluster"
-	"github.com/openshift/installer/pkg/asset/ignition/bootstrap"
-	"github.com/openshift/installer/pkg/asset/ignition/machine"
-	"github.com/openshift/installer/pkg/asset/installconfig"
-	"github.com/openshift/installer/pkg/asset/kubeconfig"
-	"github.com/openshift/installer/pkg/asset/manifests"
-	"github.com/openshift/installer/pkg/asset/metadata"
-	"github.com/openshift/installer/pkg/destroy"
-	_ "github.com/openshift/installer/pkg/destroy/libvirt"
-	"github.com/openshift/installer/pkg/terraform"
+	"github.com/spf13/cobra"
 )
 
 var (
-	installConfigCommand   = kingpin.Command("install-config", "Generate the Install Config asset")
-	ignitionConfigsCommand = kingpin.Command("ignition-configs", "Generate the Ignition Config assets")
-	manifestsCommand       = kingpin.Command("manifests", "Generate the Kubernetes manifests")
-	clusterCommand         = kingpin.Command("cluster", "Create an OpenShift cluster")
-	versionCommand         = kingpin.Command("version", "Print version information and exit")
-
-	destroyCommand = kingpin.Command("destroy-cluster", "Destroy an OpenShift cluster")
+	version = "was not built correctly" // set in hack/build.sh
```

Comment:
This shoiuld probably move to `version.go`.

Reply:
sure.